components/search/CustomHits.tsx

```diff
@@ -1,28 +1,21 @@
-import { connectStateResults } from "react-instantsearch-dom";
-import { SearchState, SearchResults } from "react-instantsearch-core";
+import { useInstantSearch } from 'react-instantsearch';
 import { WorkflowCards } from "../WorkflowCard";
 import { Workflow } from "warp-workflows";
 
-function Hits({
-  searchState,
-  searchResults,
-  children,
-}: {
-  searchState: SearchState;
-  searchResults: SearchResults;
-  children: React.ReactNode;
-}) {
-  if (!searchState.query || searchState.query.length == 0) {
+function Hits({ children }: { children: React.ReactNode }) {
+  const { results } = useInstantSearch();
```

Comment:
`connectStateResults` was removed in `react-instantsearch` v7. They [recommend](https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/react/#connectstateresults) getting the same information about state/results from this `useInstantSearch()` hook instead.